src/datatable.js

```diff
@@ -204,6 +231,10 @@ export class DataTable {
   }
 
   doSearch() {
+    if (this.offlineMode) {
+      return;
```

Comment:
If we have a reference the datatable ViewModel (view-model.ref="..."), it can be called. So just to be safe...